### DISCUSSION TITLE
bfe怎么增加https证书

### DISCUSSION THREAD

**Discussion by yangfei91** [bfenetworks/bfe#1086] [Q&A (问题和回答)]
bfe怎么增加https证书
在dashboard中添加了证书，好像不生效。

bfe转发转发https，我加了证书识别不了： error: CLIENT_TLS_HANDSHAKE,remote error: unknown certificate

**Comment by kwanhur** [comment]
1. 建议用 `openssl` 工具校验证书是否正确
2. 确认未生效，可以触发一下 `reload` 动作
3. 建议给一个可复现的最小 demo
